assets/styles/components/_comment.scss

```diff
@@ -171,6 +171,7 @@ $comment-margin-sm: .3rem;
     figure img {
       margin: .5rem 0;
       max-width: 100%;
+      border: var(--kbin-avatar-border);
```

Comment:
I'm slightly curious what situations you weren't seeing this in. It looks like it goes away at breakpoint < sm but looking at your instance that still might be true

Reply:
oops perhaps this one shouldn't be here, I think I should remove it

but basically this applies to image attached in comments, especially in microblog section, as I find it a bit odd that the image in top level post get to have this border but the following replies don't 

<details><summary>some example from fedia.io</summary>
<p>

https://fedia.io/m/random/p/239118/WTF

top level post:
![image](https://github.com/MbinOrg/mbin/assets/20770492/4a095e4b-9c50-4d99-a5ae-3730bc76e0fd)

comments:
![image](https://github.com/MbinOrg/mbin/assets/20770492/7c129b25-23e1-48f0-aa05-1a0bcfffda91)

</p>
</details>